index.html

```diff
@@ -139,12 +139,12 @@
         w3cid: 46729
       },
       {
-        name: "Jonathan Holt, DO, MS",
-        url: "https://www.linkedin.com/in/jonathan-holt-do-ms",
-        company: "ConsenSys Health",
-        companyURL: "https://www.consensyshealth.com",
-        w3cid: 95341
-      }
+      name: "Jonathan Holt, DO, MS", 
```

Comment:
Please don't add yourself as an Editor or Author of a spec. Those decisions are WG decisions and happen toward the end, when we can holistically look at who has contributed what to the specification.

Reply:
Given the amount of work I've put into this document, it only seems fair.

Reply:
Given the amount that I have contributed, only seems fair.

Reply:
Please take a look at what others in the group have contributed to the specification... there are many others that are not listed that have (arguably) contributed more: https://github.com/w3c/did-core/graphs/contributors?from=2017-07-02&to=2020-10-08&type=c

The goal here is that everyone that should be recognized for contributing to the specification will be recognized in time, and doing so is a WG decision after (ideally) taking a data-based approach to contributions across all areas (issues, PRs, calls, etc.). We typically do this right before we publish the Proposed Recommendation (many months from now). Until then, adding and removing people from the authors list tends to lead to a lot of gnashing of teeth.

If you feel strongly about this, you could always escalate it to the Chairs to get a Chair opinion on the matter.

Reply:
@brentzundel @burnburn for future reference, if we retain CBOR, jonathan holt should be listed as a contributor... potentially he should regardless, given how little contribution this note has seen from WG members.

Reply:
@OR13 🙏,  that means a lot!

Reply:
Yes, /at that time/ we should add @jonnycrunch ... not before. If we're going to start adding people to Editors and authors, we're going to have to discuss other people as well... which will block this PR.

Please remove the author credit for now to unblock the PR, please.

Reply:
@msporny I'm actually already listed as a contributor,  this PR only updates my affiliation.  Given the amount that I have contributed, it seems only fair.

Reply:
*sigh* -- I missed that. Looks that it happened here: https://github.com/w3c/did-core/commit/7a4bac4a3da280c03961c960030aea527e4135fa ... which never should have gotten into the spec.

I had asked that you remove your name at that point as well and thought you had done that:

https://github.com/w3c/did-core/pull/282/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R127-R132

 I'll raise the issue with the chairs and see how they want to proceed.

Reply:
@jonnycrunch Manu is right that individual contributors should NEVER add themselves to the author/editor/etc. lists.  These determinations are complex and can best be handled as we go to CR or even PR.  @manu , since Jonathan's name is currently there please update his affiliation. @jonnycrunch understand that the lists of editors, authors, and contributors will be updated later and your name may shift to a different list or drop off as appropriate at that time.  I am speaking in my official capacity as Co-chair.

Reply:
@burnburn wrote:
> @msporny, since Jonathan's name is currently there please update his affiliation.

Acknowledged, will do.

Reply:
I second what @burnburn has said.
The final determination of who will be listed as an author/editor/etc. has yet to be made and is complex. The editors and chairs will be the ones to make the determination, which will be based primarily on data-driven evaluation of contributions.